tagmanager.config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "development": {
```

Comment:
should we have this in repo?

Reply:
Yup, it's not sensitive information. It has no value other than telling the config which Tag Manager environment to use.